File: modul_2/PRAK205-2510817220008-NamiraHumaira.py
import math
angka = []
inputan = input().split()
angka.extend(inputan)
while(len(angka) < 2):
    inputan = input().split()
    angka.extend(inputan)
a = float(angka[0])
b = float(angka[1])
c = math.sqrt((b * b) - (a * a))
keliling = a + b + c
luas = 0.5 * c * a
print(" ")
print(f"Alas = {c:.0f} cm")
print(f"Tinggi = {a:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2\n")

File: modul_2/PRAK205-2510817220008-NamiraHumaira.c
#include <stdio.h> 
#include <math.h>
int main()
{
    int Aa, Bb, Cc, ti, kel, lu; 
printf("\n");
scanf("%d %d", &Aa, &Bb);
Cc =sqrt(pow((Bb),2)-pow((Aa),2)); 
kel =Aa + Bb + Cc;
lu = Aa * Cc / 2; 
printf("Alas = %d \n", Cc); 
printf("Tinggi = %d \n", Aa);
printf("Keliling = %d \n", kel); 
printf("Luas = %d cm^2\n", lu);
return 0;
}